oe_theme.theme

```diff
@@ -1242,11 +1242,27 @@ function oe_theme_preprocess_block__system_branding_block(&$variables) {
   $route_options = ['absolute' => TRUE];
   $variables['site_logo_href'] = Url::fromRoute('<front>', [], $route_options)->toString();
 
-  // Ensure the site logo falls back to english version if the language is
-  // not EU language.
+  // Provide correct icon for non-eu languages.
   $language_code = \Drupal::languageManager()->getCurrentLanguage()->getId();
   if (!EuropeanUnionLanguages::hasLanguage($language_code)) {
-    $variables['current_language_id'] = 'en';
+    $language_icon_mapping = [
+      'nb' => 'no',
+      'tr' => 'tu',
+    ];
+    if (isset($language_icon_mapping[$language_code])) {
+      // Set correct icon for Norwegian and Turkish languages.
+      $variables['current_language_id'] = $language_icon_mapping[$language_code];
+    }
+    else {
```

Comment:
I don't think that we should have here `else`, otherwise we will drop checking if related logo.svg exists for Turkish and Norwegian languages.
These 2 languages should be checked as other languages non-eu languages.

Reply:
We don't test that logos for European languages exist because we know that they are.
So the same logic that logos for Norwegian and Turkish exist - ECL provides them.
`else` is used as a fallback since site can have any other language that isn't determined as European or supported non-European.

Reply:
In this case, we will rely on ECL but anyway exist a chance that logo file can be removed/renamed intentionally or by mistake.